src/content/documentation/develop/debugging.md

```diff
@@ -70,6 +70,8 @@ To debug your extension, you'll use:
 - [**Debugger**](https://developer.mozilla.org/docs/Tools/Debugger) to set breakpoints and watchpoints in your extension's JavaScript, and examine and modify the extension's state.
 - [**Storage**](https://developer.mozilla.org/en-US/docs/Tools/Storage_Inspector) to view details of data stored by the extension.
 
+Some logs do not appear in the console of this Toolbox. There is an overview of all relevant places to look for messages at [Viewing log output](#viewing_log_output).
```

Comment:
Nit, phrased as "Some logs do not" does sound a bit "mysterious" :-), maybe we may phrase it differently, something like `Be aware that not all logs appears in the console panel of this Toolbox, look to the section "Viewing log output" for an overview of all the relevant places to look for logs.`.

Reply:
I've replaced the first sentence with `This console shows messages from the extension's background script; other logs may appear elsewhere.`.

Reply:
@Rob--W that's not entire true, it shows the logs from all extension pages not just the background page.

Reply:
Updated and added `and extension pages`

Reply:
ty!